meson.build

```diff
@@ -316,7 +315,3 @@ summary({
 	'man-pages': scdoc.found(),
 }, bool_yn: true)
 
-if not wlroots_features['libseat']
-	warning('The sway binary must be setuid when compiled without (e)logind or libseat')
```

Comment:
We loose this warning when libseat is compiled with the builtin backend. I wonder if it's a big deal. Probably not.

Reply:
I guess we could probe pkgconfig in the future.